client-java/BUILD

```diff
@@ -60,4 +60,23 @@ java_library(
         "//dependencies/maven/artifacts/org/slf4j:slf4j-api",
         "//dependencies/maven/artifacts/org/slf4j:slf4j-simple",
     ]
+)
+
+java_test(
+    name = "TransactionTest",
+    srcs = ["src/test/java/ai/grakn/client/TransactionTest.java","src/test/java/ai/grakn/client/ServerRPCMock.java"],
+    deps = [":client-java"] + ["//grakn-graql"] +[
+        "//dependencies/maven/artifacts/org/hamcrest:hamcrest-library",
+        "//dependencies/maven/artifacts/org/mockito:mockito-core",
+        "//dependencies/maven/artifacts/io/grpc:grpc-testing",
+        "//dependencies/maven/artifacts/io/grpc:grpc-stub",
+        "//dependencies/maven/artifacts/io/grpc:grpc-core",
+        "//dependencies/maven/artifacts/com/google/guava:guava",
+    ],
+    size = "small",
+)
+
+test_suite(
+    name = "unit_tests",
```

Comment:
Do unit tests need to be named like this, `unit_tests`, with underscores, @marco-scoppetta ?